### ISSUE TITLE
Inline checkboxes/radios that line wrap aren't horizontally aligned

### ISSUE THREAD

**Issue by surjithctly** [twbs/bootstrap#19638]
As title says, Inline checkboxes and radios are broken in mobile screens.

Demo : https://jsfiddle.net/mhyoffw9/

Image: 
![inline-checkbox](https://cloud.githubusercontent.com/assets/1884712/14114712/b2c797a2-f5f5-11e5-8bd2-7eb3b7b3e2e5.png)

**Comment by twbs-lmvtfy** [comment]
Hi @surjithctly!

You appear to have posted a live example (https://fiddle.jshell.net/mhyoffw9/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W005](https://github.com/twbs/bootlint/wiki/W005): Unable to locate jQuery, which is required for Bootstrap's JavaScript plugins to work

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by RyanZim** [comment]
Possible Duplicate of #18582.

**Comment by mdo** [comment]
I don't think there's anyway we can deal with this automatically for you. I can't nuke the margin between those elements for just mobile because there are instances where the inline controls will indeed be on a single line. Best option I can give you is use the margin utilities to override it or consider custom styles.
